config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -0,0 +1,76 @@
+presets:
+- labels:
+    preset-azure-acsengine: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: CLOUD_CONFIG
+    value: random 
```

Comment:
out of curiosity I'd love to see some of these commented with why they are set / what they do :-)

Reply:
https://github.com/kubernetes/kubernetes/blob/master/hack/ginkgo-e2e.sh#L108

This is a ugly, horrible way to avoid this. We don't use it in any way in this scenario, yet it needs to be set to something for tests to actually run.